Formula/bloaty.rb

```diff
@@ -1,7 +1,6 @@
 class Bloaty < Formula
   desc "Size profiler for binaries"
   homepage "https://github.com/google/bloaty"
-  # TODO: Check if we can use unversioned `protobuf` at version bump
```

Comment:
I've omitted the `revision` bump since this will likely get one in #132437 soon-ish.